Kitodo-DataEditor/src/test/java/org/kitodo/dataeditor/ruleset/RulesetManagementIT.java

```diff
@@ -704,6 +706,15 @@ public void testFieldsWithMinOccursGreaterZeroAreAlwaysShown() throws IOExceptio
         assertThat(ids(mvwviList), contains("test1", "test2", "test2", "test2options"));
     }
 
+    @Test
+    public void testGettingOfSpecialFields() throws Exception {
+        RulesetManagementInterface underTest = new RulesetManagement();
+        underTest.load(new File("src/test/resources/testAnExtensiveRulesetCanBeLoaded.xml"));
+
+        assertThat(underTest.getIdsOfKeysForSpecialField(SpecialField.TITLE), contains("TitleDocMain"));
+        assertThat(underTest.getIdsOfKeysForSpecialField(SpecialField.AUTHOR_LAST_NAME), contains("Person/LastName"));
+    }
```

Comment:
please test also for not existing uses
test, that the result is exactly the ids not only "contains" because if the method always returns all ids it will still be true
please test for multiple uses separated by comma in one key

Reply:
Added tests for not-existing uses and multiple uses of one key. `contains()` here defines equality, so if there would be more elements returned, this would cause the test to fail. Multiple uses are to be separated by space, not comma. This is how the [NMTOKENS is defined](https://www.w3.org/TR/xmlschema11-2/#NMTOKENS) in XML.